stdlib/public/core/Stride.swift.gyb

```diff
@@ -62,30 +62,53 @@ public func == <T : Strideable>(x: T, y: T) -> Bool {
   return x.distance(to: y) == 0
 }
 
-public func + <T : Strideable>(lhs: T, rhs: T.Stride) -> T {
+% for Base in ['Strideable', 'SignedInteger']:
+%   deprecated = Base == 'SignedInteger'
+
+%   if deprecated:
+@available(swift, deprecated: 3.1, obsoleted: 4.0, message: "Mixtype addition is deprecated. Please use explicit type conversion.")
```

Comment:
Can you put "Mixed-type" instead of "Mixtype"? I don't think the latter is a standard term.